data-migration/2.0-to-3.0/migrate.js

```diff
@@ -0,0 +1,39 @@
+/* global db */
+
+const migrate = () => {
+  print('Migrating data 2.0 to 3.0')
+
+  const query = {}
+
+  // Note: keys 'dataManagementPlanUrl', 'homeUrl', 'imageCaption', and
+  // 'softwareSustainabilityPlanUrl' are purposely not set, as they are
+  // not required. Users can still set them through the admin
+  // interface if they want though.
+  const update = {
+    $set: {
+      callUrl: 'https://doi.org/FIXME',
+      codeUrl: 'https://github.com/FIXME',
+      dateEnd: '1901-01-01',
+      dateStart: '1900-01-01',
+      description: 'FIXME FIXME FIXME FIXME FIXME FIXME FIXME FIXME FIXME FIXME FIXME FIXME FIXME FIXME FIXME FIXME FIXME FIXME FIXME FIXME',
+      grantId: 'FIXME',
+      image: { data: '', mimeType: 'image/jpg' },
```

Comment:
we can add the default image (placeholder) here

Reply:
I'll make this part of https://github.com/research-software-directory/research-software-directory/pull/696#discussion_r532744870

Reply:
#710